### PR TITLE
roadmap: update to better reflect v0.36 changes

### PR DESCRIPTION
**Pull request by cmwaters**
I have attempted to update our roadmap to reflect some of the decisions made. Of note:
- Pushing soft upgrades back to v0.37
- Adding RPC Event Subscription to v0.36
- Specifying more concretely our plan to implement LibP2P

### CODE REVIEW

File: docs/roadmap/roadmap.md [hunk 1]
```diff
@@ -43,45 +43,46 @@ Added a new `EventSink` interface to allow alternatives to Tendermint's propriet
...
 
 ### Proposer-Based Timestamps
 
 Proposer-based timestamps are a replacement of [BFT time](https://docs.tendermint.com/master/spec/consensus/bft-time.html), whereby the proposer chooses a timestamp and validators vote on the block only if the timestamp is considered *timely*. This increases reliance on an accurate local clock, but in exchange makes block time more reliable and resistant to faults. This has important use cases in light clients, IBC relayers, CosmosHub inflation and enabling signature aggregation. [More](https://github.com/tendermint/tendermint/blob/master/docs/architecture/adr-071-proposer-based-timestamps.md)
 
-### Soft Upgrades
+### RPC Event Subscription
 
-We are working on a suite of tools and patterns to make it easier for both node operators and application developers to quickly and safely upgrade to newer versions of Tendermint. [More](https://github.com/tendermint/spec/pull/222)
+[More](https://github.com/tendermint/tendermint/blob/master/docs/architecture/adr-075-rpc-subscription.md)
```
Comment (cmwaters) [inline docs/roadmap/roadmap.md:54]: @creachadair do you mind adding a snippet of the work that this encompasses?

Comment (creachadair) [reply docs/roadmap/roadmap.md]: I filled out this paragraph a bit, PTAL.

**Review by creachadair — APPROVED** [review]

File: docs/roadmap/roadmap.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ order: 1
 
 # Tendermint Roadmap
 
-*Last Updated: Friday 8 October 2021*
+*Last Updated: Fridya 4 February 2022*
```
Comment (creachadair) [inline docs/roadmap/roadmap.md:7]:
```suggestion
*Last Updated: Friday 4 February 2022*
```

File: docs/roadmap/roadmap.md [hunk 3]
```diff
@@ -43,45 +43,46 @@ Added a new `EventSink` interface to allow alternatives to Tendermint's propriet
...
 
 - Remove the "legacy" P2P framework, and clean up of P2P package. [More](https://github.com/tendermint/tendermint/issues/5670)
 - Remove the global mutex from the local ABCI client to enable application-controlled concurrency. [More](https://github.com/tendermint/tendermint/issues/7073)
-- Enable P2P support for light clients
-- Node orchestration of services + Node initialization and composibility
+- Improve life cycle management of a node and its reactors.
 - Remove redundancy in several data structures. Remove unused components such as the block sync v2 reactor, gRPC in the RPC layer, and the socket-based remote signer.
-- Improve node visibility by introducing more metrics
+- Improve node visibility through the introduction of more metrics
+- Migrating locally configured consensus timeouts to global consensus params. [More](https://github.com/tendermint/tendermint/blob/master/docs/architecture/adr-074-timeout-params.md)
```
Comment (creachadair) [inline docs/roadmap/roadmap.md:63]:
```suggestion
- Migrating locally configured consensus timeouts to global consensus parameters. [More](https://github.com/tendermint/tendermint/blob/master/docs/architecture/adr-074-timeout-params.md)
```

**Review by creachadair — APPROVED** [review]

File: docs/roadmap/roadmap.md (hunk 2 above)
Comment (cmwaters) [reply docs/roadmap/roadmap.md:7]: Good catch

**Pull request merged**